analysis/topEFT/topeft.py

```diff
@@ -246,7 +247,7 @@ def process(self, events):
         l1 = l_fo_conept_sorted_padded[:,1]
         l2 = l_fo_conept_sorted_padded[:,2]
 
-        print("The number of events passing FO 2l, 3l, and 4l selection:", ak.num(events[events.is2l],axis=0),ak.num(events[events.is3l],axis=0),ak.num(events[events.is4l],axis=0))
+        #print("The number of events passing FO 2l, 3l, and 4l selection:", ak.num(events[events.is2l],axis=0),ak.num(events[events.is3l],axis=0),ak.num(events[events.is4l],axis=0))
```

Comment:
Should we delete this one, or will we eventually uncomment it again?

Reply:
Hmm, I had actually not noticed that we had commented it out. I think it does not hurt to leave it in for now, as it can be useful sometimes when running short tests. So I'd be fine with either uncommenting it, or just leaving it in commented out, whichever you'd suggest.

Reply:
I  think we can uncomment it.

Reply:
Ok, done.

Reply:
Thanks!